qcodes/instrument/parameter.py

```diff
@@ -257,8 +257,8 @@ def __init__(self, name: str,
             warnings.warn(f"_BaseParameter got unexpected kwargs: {kwargs}."
                           f" These are unused and will be discarded. This"
                           f" will be an error in the future.")
-        self.name = str(name)
-        self.short_name = str(name)
```

Comment:
I don't think we need an interal property for both short_name and name ? Could we not just have name point to _short_name

Reply:
Yes. Done as suggested